### PR TITLE
Add mutation observer for element appear checks

### PR DESCRIPTION
**Pull request by peterszerzo**

### CODE REVIEW

File: packages/core/index.html [hunk 1]
```diff
@@ -58,9 +60,19 @@ <h1>My App2</h1>
...
-    elem.addEventListener("click", e => { console.log("clicked", e.currentTarget) }))
+  document.querySelectorAll(".click").forEach((elem) =>
+    elem.addEventListener("click", (e) => {
+      // console.log("clicked", e.currentTarget);
+    })
+  );
+
+  document.querySelector("#two").addEventListener("click", (e) => {
+    const newElem = document.createElement("div");
+    newElem.innerHTML = "<p>abcd</p>";
```
Comment (RomanistNLX) [inline packages/core/index.html:75]: you could have written here something nice (or at least "hello world")

**Review by RomanistNLX — APPROVED** [review]
:+1:

File: packages/core/index.html (hunk 1 above)
Comment (peterszerzo) [reply packages/core/index.html:75]: Yes I could have :)

Comment (peterszerzo) [reply packages/core/index.html:75]: Treating it as kind of a throwaway file at the moment, we can make it nicer.

**Pull request merged**